snap7/server/__init__.py

```diff
@@ -177,17 +194,17 @@ def start(self, tcpport: int = 102):
         return self._lib.Srv_Start(self._s7_server)
 
     @error_wrap
-    def stop(self):
+    def stop(self) -> int:
         """Stop the server."""
         logger.info("stopping server")
         return self._lib.Srv_Stop(self._s7_server)
 
-    def destroy(self) -> Optional[int]:
+    def destroy(self) -> None:
         """Destroy the server."""
         logger.info("destroying server")
         if self._lib and self._s7_server is not None:
             return self._lib.Srv_Destroy(byref(self._s7_server))
-        self._s7_server = None
+        self._s7_server = None  # type: ignore[assignment]
```

Comment:
also this one should be optional type

Reply:
It's the same as https://github.com/gijzelaerr/python-snap7/pull/516#discussion_r1623571102